### PR TITLE
[TASK] Documentation: Nginx reverse proxy, Virtual_* variable

### PR DESCRIPTION
**Pull request by Cyb10101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,11 @@ services:
       - ~/.gitconfig:/home/application/.gitconfig
 
     environment:
+      # See for VIRTUAL_* in documentation: Nginx Reverse Proxy
       - VIRTUAL_HOST=~^docker-website-[a-z]+\.vm$$
+      #- VIRTUAL_PROTO=https
```
Comment (Kanti) [inline README.md]: in what case is this needed?

File: docs/nginx-reverse-proxy.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Here is an information about the VIRTUAL_* variable clarified.
+
+## VIRTUAL_HOST: Virtual host (your domain)
+
+So that the nginx reverse proxy knows which domain belongs to which Docker container there is the VIRTUAL_HOST variable.
+
+##### Variant A
+
+You can simply specify this domain directly without regular expressions. See variant A.
```
Comment (Kanti) [inline docs/nginx-reverse-proxy.md]:
```suggestion
You can specify this domain directly without regular expressions. See variant A.
```

As described more clearly here: https://css-tricks.com/words-avoid-educational-writing/
Words like `simply` shouldn't be present in Documentation or any code related writings.

File: README.md (hunk 1 above)
Comment (Cyb10101) [reply README.md]: Directly communicated.

File: docs/nginx-reverse-proxy.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+You use variant C for the Symfony website, because domain.vm and www.domain.vm should be used.
+
+For the forum you use variant D, so you have forum.domain.vm.
+
+Finally, I can say that variant C or D is the better choice in most cases.
+But since every system is different, you can of course do what suits you.
+
+File docker-compose.yml:
+
+```yaml
```
Comment (Kanti) [inline docs/nginx-reverse-proxy.md]: if the example file is on top I don't have to scroll so often.

File: docs/nginx-reverse-proxy.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+# Nginx Reverse Proxy
+
+As already mentioned, this project relies on [pluswerk/docker-global](https://github.com/pluswerk/docker-global).
+
+Here is an information about the VIRTUAL_* variable clarified.
+
+## VIRTUAL_HOST: Virtual host (your domain)
+
+So that the nginx reverse proxy knows which domain belongs to which Docker container there is the VIRTUAL_HOST variable.
```
Comment (ChrisB9) [inline docs/nginx-reverse-proxy.md]:
```suggestion
Through the VIRTUAL_HOST variable the nginx reverse proxy knows which domain belongs to which Docker container.
```

File: docs/nginx-reverse-proxy.md (hunk 2 above)
Comment (ChrisB9) [inline docs/nginx-reverse-proxy.md]:
```suggestion
You can specify this domain directly without regular expressions. Look at variant A.
```

File: docs/nginx-reverse-proxy.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+##### Variant A
+
+You can simply specify this domain directly without regular expressions. See variant A.
+
+##### Regular expression information
+
+However, I would recommend a bit of regular expressions.
```
Comment (ChrisB9) [inline docs/nginx-reverse-proxy.md]:
```suggestion
We do recommend the use of regular expressions.
```

File: docs/nginx-reverse-proxy.md (hunk 3 above)
Comment (ChrisB9) [reply docs/nginx-reverse-proxy.md]: i think so too, the examples should go directly under the variants

Comment (Cyb10101) [reply docs/nginx-reverse-proxy.md]: Done

**Pull request merged**